src/renderer/src/hooks/useNeonBackup.ts

```diff
@@ -101,6 +103,19 @@ export const backupFileSchema = zod.object({
   data: zod.string(),
 })
 
+const fixSkinAccount = (skin?: TSkin): TSkin => {
+  if (!skin) return { id: ACCOUNT_COLOR_SKINS[UtilsHelper.getRandomNumber(7)].id, type: 'color' }
+  if (skin.type !== 'color') return skin
+
+  const foundColorSkin = ACCOUNT_COLOR_SKINS.find(
+    ({ color }) => color.toLowerCase().replace(/[bg\-[|\]]/g, '') === skin.id.toLowerCase()
+  )
+
+  if (foundColorSkin) skin.id = foundColorSkin.id
+
+  return skin
+}
```

Comment:
I think it should be simpler to check here if the skin is hexadecimal and, if so, generate a new random color. This way, we don't need to change anything in the AccountIcon component.

```suggestion

const fixSkinAccount = (skin?: TSkin): TSkin => {
  if (!skin || UtilsHelper.isHexadecimal(skin.id) ) return { id: ACCOUNT_COLOR_SKINS[UtilsHelper.getRandomNumber(7)].id, type: 'color' }

  return skin
}
```